doc/user_doc/vic_installation/vch_installer_options.md

```diff
@@ -60,6 +60,8 @@ Short name: `-i`
 
 The datastore in which to store container image files. When you deploy a virtual container host, `vic-machine` creates a folder named `VIC` on the target datastore,  in which to store all of the container images that you pull into a virtual container host. The `vic-machine` utility also places the VM files for the virtual container host in the datastore that you designate as the image store, in a folder that has the same name as the virtual container host. 
 
+If you are deploying the virtual container host to a vCenter Server cluster, the datastore that you designate in the `image-store` option must be shared by all of the ESXi hosts in the cluster. Using non-shared datastores is possible, but limits the use of vSphere features such as DRS and High Availability.
```

Comment:
Our current requirement is to have multiple hosts connect to the shared datastore, instead of all hosts.
And cause VIC accepts multiple datastore configuration for image, volume and container in the future. User need to make sure there are at least two hosts connected to all of those datastores.
As it's kind of complicated to describe, maybe your words are good. Just let you know the details.